app/controllers/families_controller.rb

```diff
@@ -143,6 +143,22 @@ def schema
     render xml: Family.columns.map { |c| {name: c.name, type: c.type} }
   end
 
+  def directory_map
+  end
+
+  def map_view_details
+    mapable_families = Family.all.select{|f| f.mapable?}
+    map_view_details = []
+    mapable_families.each do |family|
+      map_view_details << family.attributes.select{|key, value| ['longitude', 'latitude', 'name'].include?(key)}
```

Comment:
Unused block argument - `value`. If it's necessary, use `_` or `_value` as an argument name to indicate that it won't be used.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.